frontend/src/components/Cart/CartSidebarItem.vue

```diff
@@ -19,8 +19,36 @@
 				font="text-md"
 			/>
 
-			<div class="flex flex-col justify-between w-full">
+			<div class="flex flex-col justify-between w-full relative">
 				<div class="text-sm font-semibold">{{ row.item_name }}</div>
+				<!-- Delete Menu Only Desktop Screens -->
+				<div class="cursor-pointer absolute -top-1 -right-1">
+					<Popover>
+						<template #target="{ togglePopover }">
+							<Button
+								:variant="'ghost'"
+								theme="gray"
+								size="sm"
+								@click.stop="togglePopover()"
+								class="w-[10px]"
+							>
+								<EllipsisVertical size="16px" />
```

Comment:
What is size property? Use CSS/Tailwind Classes for height/width

Reply:
Ye jo hum icon library use kar rahai hain  wo hi kuch attributes provide karti hain
For Example: size, stroke and etc..